hrqb/tasks/employee_appointments.py

```diff
@@ -109,7 +109,6 @@ def get_dataframe(self) -> pd.DataFrame:
         )
 
         fields = {
-            "hr_appt_key": "HR Appointment Key",
```

Comment:
This is an example of a data warehouse foreign key that was _assumed_ to be enduring, but actually changes day-to-day.

QB tables have slowly moved into an explicit `Key` field approach, that is written by HRQBClient.  This field was leftover and removing it does not remove any meaningful data from QB.